src/howtos/install_epinio_on_rancher_desktop.md

```diff
@@ -37,6 +40,7 @@ The Epinio installation is pretty much identical on Linux, MacOS and Windows:
 
 2. Download the [latest version](https://github.com/epinio/epinio/releases) of Epinio binary and copy it on your client. It is recommended to add it to the `PATH` variable (`export PATH=<epinio-binary-path>:$PATH` on Linux/MacOS with Bash and on Windows please follow [Kevin Berg's article](https://medium.com/@kevinmarkvi/how-to-add-executables-to-your-path-in-windows-5ffa4ce61a53)).
```

Comment:
Better link to this page? https://docs.epinio.io/installation/install_epinio_cli.html

Reply:
Done, but this step is (will be) optional to me, especially with the standalone UI in.